roles/network/templates/hostapd/iiab-hostapd.conf.j2

```diff
@@ -1,6 +1,10 @@
 # Basic configuration
 
-interface={{ discovered_wireless_iface }}
+{%if wifi_up_down  %}
+interface=ap0
+{% else %}
+interface={{ iiab_wireless_lan_iface }}
+{% endif %}
```

Comment:
as above wifi only installs would not have iiab_wireless_lan_iface present I would suggest using discovered_wireless_iface